experimental/gittuf/repository_test.go

```diff
@@ -54,3 +54,29 @@ func TestUnauthorizedKey(t *testing.T) {
 		assert.ErrorIs(t, err, ErrUnauthorizedKey)
 	})
 }
+
+func TestGetGitRepository(t *testing.T) {
+	tmpDir := t.TempDir()
+	repo := gitinterface.CreateTestGitRepository(t, tmpDir, false)
+
+	r := &Repository{r: repo}
+	result := r.GetGitRepository()
+
+	assert.Equal(t, repo, result, "Error: GetGitRepository does not return repo")
+}
+
+func TestLoadRepository_NoRepo(t *testing.T) {
```

Comment:
you could make this just `TestLoadRepository` and have a `t.Run` for a no repo failure case. Also a `t.Run` for a successful load.

Reply:
Ah `TestLoadRepository exists, I'd move everything in that into a `t.Run` for the successful load case and add this as a `t.Run` for the expected failure case.